app/vmalert/group.go

```diff
@@ -71,6 +74,12 @@ func newGroupMetrics(g *Group) *groupMetrics {
 		g.mu.RUnlock()
 		return i
 	})
+	m.iterationIntervalOffset = utils.GetOrCreateGauge(fmt.Sprintf(`vmalert_iteration_interval_offset_seconds{%s}`, labels), func() float64 {
```

Comment:
I think, that metric makes sense only if g.IntervalOffset is explicitly set. Otherwise it'll show -1 value. It could be fixed by added check for g.IntervalOffset >=0

Reply:
@spotifyprism what is your intention for using this metric? Is it just informative or actionable?